ubuntu/16.04/install.sh

```diff
@@ -83,20 +83,20 @@ systemctl enable xrdp.service
 systemctl enable xrdp-sesman.service
 
 # Configure the policy xrdp session
-bash -c 'cat >/etc/polkit-1/localauthority.conf.d/02-allow-colord.conf <<EOF
-
-polkit.addRule(function(action, subject) {
-    if ((action.id == "org.freedesktop.color-manager.create-device" ||
-         action.id == "org.freedesktop.color-manager.modify-profile" ||
-         action.id == "org.freedesktop.color-manager.delete-device" ||
-         action.id == "org.freedesktop.color-manager.create-profile" ||
-         action.id == "org.freedesktop.color-manager.modify-profile" ||
-         action.id == "org.freedesktop.color-manager.delete-profile") &&
-        subject.isInGroup("users"))
-    {
-        return polkit.Result.YES;
-    }
-});
+
+# polkit policy definition language changes depending on its version. See issue #61
+if [[ "$(pkaction --version | sed -E 's/^[[:alnum:] ]*([[:digit:]]+.*)/\1/' - )" != '0.105' ]]; then
+    echo "Error: Policy rule specification probably invalid. Expected version: 0.105 detected $(pkaction --version)." >&2
+    exit 1
+fi
+
+bash -c 'cat > /etc/polkit-1/localauthority/50-local.d/45-allow-colord.pkla <<EOF
```

Comment:
This is running as root so can just do `cat > /etc/polkit... <<EOF`

Reply:
Will apply change requested by @jterry75.